gallery_dl/ffprobe.py

```diff
@@ -57,8 +57,7 @@ def get_video_length(obj, url):
             )
             data = json.loads(result.stdout)
         except subprocess.CalledProcessError as e:
-            msg = "ffprobe failed: " + e
-            print(e)
+            msg = "ffprobe failed: " + str(e)
```

Comment:
you removed the `print` as well

Reply:
This was intentional since logging is already done in the loop in line 44 using the logger object.